admittance_controller/src/admittance_controller.cpp

```diff
@@ -27,6 +27,23 @@
 
 namespace admittance_controller
 {
+
+static geometry_msgs::msg::Wrench add_wrenches(
```

Comment:
```suggestion
geometry_msgs::msg::Wrench add_wrenches(
```

I don't think static is necessary here

Reply:
Depends on the desired visibility - I tend to make all freestanding functions static per default so that they are not visible to any other compilation units.

Reply:
If you don't make them static,  the modern compilers optimize them still, it's called return value optimization

Reply:
If we do not declare them static we basically tell the compiler that somewhere else in the code we may write a definition for that function and call it. Also from my point of view: If I do not declare a function in a header file I make it static as I do not want to expose it outside of the compilation unit.

Reply:
We've been using anonymous namespaces to achieve this which avoids mixing in the semantics of static functions/objects.